types/three/examples/jsm/shaders/SSRShader.d.ts

```diff
@@ -0,0 +1,57 @@
+import { Matrix4, Vector2, Texture } from '../../../src/Three';
+/**
+ * References:
+ * https://lettier.github.io/3d-game-shaders-for-beginners/screen-space-reflection.html
+ */
+
+export interface SSRShader {
```

Comment:
I believe we actually need to export the objects from this file as well, not just the interfaces.

Reply:
I'm not sure what you mean?

Reply:
Something like:

```
export const SSRShader: SSRShader;
export const SSRDepthShader: SSRDepthShader;
export const SSRBlurShader: SSRBlurShader;
```